packages/paste-website/stories/TokenCard.stories.tsx

```diff
@@ -91,6 +91,7 @@ const Template: StoryFn<typeof TokenCard> = ({name, category, useCamelCase}) =>
         textColorInverse={textColorInverse}
         borderColor={borderColor}
         useCamelCase={useCamelCase}
+        text_contrast_pairing={text_contrast_pairing}
```

Comment:
Question: why is this prop snake case? Could it be like the others?

Reply:
It totally could, essentially all this stuff is typed from the Tokens Object itself, and that's how it's named on the object, which is unfortunate. it's just lazy typing  to be able to pass the object around without too much hassle and map the props to the object keys